go/vt/servenv/version.go

```diff
@@ -1,3 +1,3 @@
 package servenv
 
-const versionName = "10.0.1-SNAPSHOT"
+const versionName = "10.0.2-SNAPSHOT"
```

Comment:
Do we change this for every patch release? I thought it was going to stay at 10.0.0-SNAPSHOT.

Reply:
Nvm, this makes sense.